### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by CodebyBaki** [samueloreoluwa4/Machine-learning-projects#3]
The size of the validation set is small. 0.2 is better, it Should be 60% train set, 20% validation set and 20% test set.
Also why cross validating on the training set even before trying the model on the validation set

**Comment by samueloreoluwa4** [comment]
Fixed issues
